features/step_definitions/common_steps.rb

```diff
@@ -483,7 +483,12 @@ def enable_pool(target)
 end
 
 When(/^I register using "([^"]*)" key$/) do |arg1|
-  regurl = "http://#{$server_ip}/XMLRPC"
+  # register to the proxy, or if none, to the server
```

Comment:
http://www.thechrisoshow.com/2009/02/16/using-guard-clauses-in-your-ruby-code/

Reply:
I did not touch that one yet. But it might come, for sure.

Reply:
replaced with a ternary